tests/TestCase/Db/Adapter/DefaultPdoAdapterTrait.php

```diff
@@ -0,0 +1,180 @@
+<?php
+declare(strict_types=1);
+
+namespace Migrations\Test\TestCase\Db\Adapter;
+
+use Migrations\Db\AlterInstructions;
+use Migrations\Db\Literal;
+use Migrations\Db\Table\Column;
+use Migrations\Db\Table\ForeignKey;
+use Migrations\Db\Table\Index;
+use Migrations\Db\Table\Table;
+
+trait DefaultPdoAdapterTrait
```

Comment:
We have a few places where we need a custom PDO instance which only returns specific values in specific cases. So I just added this trait to more easily make it possible to "mock" those instances without having to re-define all those methods.